Tella/Scenes/Uwazi/ViewModel/DownloadedTemplatesVM.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  DownloadedTemplatesVM.swift
+//  Tella
+//
+//  Created by Gustavo on 27/09/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class DownloadedTemplatesViewModel: ObservableObject {
+    
+    var mainAppModel : MainAppModel
+    
+    @Published private var downloadedTemplates : [CollectedTemplate] = []
+    
+    @Published var templateCardsViewModel : [TemplateCardViewModel] = []
+    
+    @Published var isLoading: Bool = false
+    @Published var serverName : String
+    
+    var server: Server
+    
+    var tellaData: TellaData? {
+        return self.mainAppModel.vaultManager.tellaData
+    }
+    
+    init(mainAppModel : MainAppModel, server: Server) {
+        
+        self.mainAppModel = mainAppModel
+        self.server = server
+        self.serverName = server.name ?? ""
+    }
+
+    
+    func getDownloadedTemplates() {
+        self.downloadedTemplates = self.getAllDownloadedTemplate() ?? []
```

Comment:
What do you think if you update this to 
` self.downloadedTemplates =  self.tellaData?.getAllUwaziTemplate()` and remove  `getAllDownloadedTemplate` function